src/Accounts/Views/Network/Contacts.cshtml

```diff
@@ -0,0 +1,157 @@
+﻿@model CommunAxiom.Accounts.ViewModels.Network.ManageViewModel
+@{
+    ViewData["Title"] = "Manage Network";
+}
+
+<div class="columns is-variable is-5">
+    <div class="column is-3">
+        <div class="columns">
+            <div class="column is-12">
+                <div class="card has-background-white">
+                    <div class="card-content">
+                        <table class="table is-hoverable">
+                            <thead>
+                                <tr>
+                                    <th colspan="2"><h5>My Network</h5></th>
```

Comment:
My network seems to be intended to work the same way across all three views, it should probably be a single partial view

Reply:
created a partial view with the card and the table. implemented in all three views